.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: java
-dist: trusty
```

Comment:
Wow. Do we know why it's so much slower on Trusty? Precise is EOL now, so it's not a terribly useful target in general.